product/reports/670_Performance by Asset Type - Middleware Servers/160_Web Metrics - hourly over last day.yaml

```diff
@@ -0,0 +1,61 @@
+---
+where_clause:
+dims:
+created_on: 2016-10-09 14:13:30.822035 Z
```

Comment:
created_on and updated_on fields are set on 2016. are those necessary in the first place? 
please check. if yes - they should be at least with an updated date :)